src/neo4j_graphrag/embeddings/mistral.py

```diff
@@ -24,7 +24,9 @@
 try:
     from mistralai import Mistral
 except ImportError:
-    Mistral = None  # type: ignore
+    # Define placeholder type for type checking
+    class Mistral:  # type: ignore
```

Comment:
Is this better compared to the previous approach? (out of curiosity)

Reply:
Not sure why this slipped in. Removed.